.github/workflows/main.yml

```diff
@@ -10,131 +10,114 @@ on:
-          if-no-files-found: error
-
-
-  run-snakemake:
-    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10
+        env:
+          POSTGRES_DB: test_aiida
+          POSTGRES_PASSWORD: ''
+          POSTGRES_HOST_AUTH_METHOD: trust
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+      rabbitmq:
+        image: rabbitmq:3.8.14
+        ports:
+          - 5672:5672
+      slurm:
+        image: xenonmiddleware/slurm:17
+        ports:
+          - 5001:22
 
     steps:
-      - name: checkout-repository
-        uses: actions/checkout@v2
-
-      - name: install-basic-deps
-        uses: ./.github/actions/install-basic-deps
-
-      - name: setup-conda-environment
-        uses: conda-incubator/setup-miniconda@v2
-        with:
-          miniforge-version: latest
-          activate-environment: simple_use_case
-          channels: conda-forge,bioconda
-
-      - name: run-workflow
-        shell: bash -l {0}
-        run: |
-          conda install snakemake
-          cd $GITHUB_WORKSPACE/simple_use_case/snakemake
-          snakemake --cores 1 --use-conda --conda-frontend conda ./paper.pdf
-
-      - name: upload-paper-artifact
-        uses: actions/upload-artifact@v2
-        with:
-          name: paper
-          path: ./simple_use_case/snakemake/paper.pdf
-          retention-days: 1
-          if-no-files-found: error
+    - name: checkout-repository
+      uses: actions/checkout@v2
+
+    - name: install-basic-deps
+      uses: ./.github/actions/install-basic-deps
+
+    - name: install-system-dependencies
+      run: |
+        sudo apt update
+        sudo apt install postgresql graphviz
+
+    - name: setup-conda-environment
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        environment-file: simple_use_case/source/envs/default_env.yaml
+        miniforge-version: latest
+        activate-environment: simple_use_case
+
+    - name: install-pypdf2
+      # It is crucial to update `setuptools` or the installation of `pymatgen` can break
+      shell: bash -l {0}
+      run:
+        conda install --channel conda-forge pypdf2 setuptools pip --yes
+
+    - name: install-aiida-core
+      working-directory: simple_use_case/aiida
+      # the shell directive is necessary to properly activate the shell
+      # see https://github.com/marketplace/actions/setup-miniconda
+      shell: bash -l {0}
+      run: |
+        git clone https://github.com/aiidateam/aiida-core.git
+        cd aiida-core
+        git checkout v2.0.0b1
+        pip install -e .
+
+    - name: install-aiida-shell
+      working-directory: simple_use_case/aiida
+      shell: bash -l {0}
+      run: |
+        git clone https://github.com/sphuber/aiida-shell.git
+        cd aiida-shell
+        git checkout v0.1.0
+        pip install -e .
+
+    - name: check-conda-env
```

Comment:
is this job and the `check-status` job left over from debugging or do these checks prevent something? I would expect that if these two jobs failed, then the pipeline would also fail without these two jobs existing? If so, could we delete them (the action is already quite long & complex)?

Reply:
Yes, these jobs were left over from debugging.